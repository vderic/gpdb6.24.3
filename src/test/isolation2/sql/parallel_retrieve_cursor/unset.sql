DROP EXTENSION gp_parallel_retrieve_cursor;
!\retcode gpconfig -r statement_timeout;
!\retcode gpstop -u;
