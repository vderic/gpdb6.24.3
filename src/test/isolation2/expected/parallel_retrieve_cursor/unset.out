DROP EXTENSION gp_parallel_retrieve_cursor;
DROP
!\retcode gpconfig -r statement_timeout;
(exited with code 0)
!\retcode gpstop -u;
(exited with code 0)
